src/test/java/io/github/christophermanahan/carnitas/HTTPResponseBuilderTest.java

```diff
@@ -0,0 +1,41 @@
+package io.github.christophermanahan.carnitas;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class HTTPResponseBuilderTest {
+    @Test
+    void buildsGETResponses() {
+        String statusCode = StatusCodes.GET;
+        String version = Constants.VERSION;
+
+        Response response = new HTTPResponseBuilder()
+          .statusCode(statusCode)
+          .version(version)
```

Comment:
I think it makes sense to set the version inside the builder as the version rarely changes

Reply:
The version is now located in the HTTPResponse class.